webrender/src/renderer.rs

```diff
@@ -3137,10 +3137,10 @@ impl<B: hal::Backend> Renderer<B> {
                                 self.device.set_scissor_rect(scissor_rect);
                             }
 
-                            #[cfg(all(not(feature = "gleam"), not(feature = "push_constants")))]
-                            let program = self.device.bound_program();
-                            #[cfg(all(not(feature = "gleam"), not(feature = "push_constants")))]
-                            self.device.set_uniforms(&program, projection);
+                            if !cfg!(feature = "gleam") && !self.device.use_push_consts {
```

Comment:
I don't think `!cfg!(feature = "gleam")` will work, you should use `cfg!(not(feature = "gleam")` since the `cfg!` macro accepts the same syntax as the `#[cfg(...)]` attribute.

Reply:
It does compile, but I agree `cfg(not(` is more readable.